src/vmspawn/vmspawn-util.c

```diff
@@ -236,3 +266,75 @@ int find_qemu_binary(char **ret_qemu_binary) {
 
         return find_executable(qemu_arch_specific, ret_qemu_binary);
 }
+
+int vsock_fix_child_cid(unsigned *machine_cid, const char *machine, int *ret_child_sock) {
+        _cleanup_close_ int vfd = -EBADF;
+        int r, i;
+        static const uint8_t sip_key[HASH_KEY_SIZE] = { 0 };
```

Comment:
while the NUL key certainly works we usually generate a proper key, for example, via `dd if=/dev/urandom bs=16 count=1 status=none | xxd -i`

Reply:
Fixed in ef3bb39e80.